pkg/registration/subscription.go

```diff
@@ -0,0 +1,14 @@
+package registration
+
+import "time"
+
+// A subscription used in the offline registration workflow
```

Comment:
Is this really specific to offline registration? If so I'd reflect it also on the naming to avoid confusion, just like we do with `OfflineCertificate` or `OfflinePayload`. Yes, it's a bit of a pity to keep adding `Offline*` to things, but I prefer that to avoid silly mistakes in the future.

Reply:
You are right. I moved it now to `SubscriptionInfo` to match the endpoint. If we in the future implement `subscription_info` endpoint in the library it will match the structure and can be reused. Fine for you?